### PR TITLE
Fix for building osu-micro-benchmarks on OS X (librt not avaialble/required)

### PR DESCRIPTION
**Pull request by pramodskumbhar**

### CODE REVIEW

File: var/spack/repos/builtin/packages/osu-micro-benchmarks/package.py [hunk 1]
```diff
@@ -56,6 +56,10 @@ def install(self, spec, prefix):
                 '--with-cuda=%s' % spec['cuda'].prefix,
             ])
 
+        # librt not available on darwin (and not require)
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/osu-micro-benchmarks/package.py]: Require*d*

File: var/spack/repos/builtin/packages/osu-micro-benchmarks/package.py [hunk 2]
```diff
@@ -56,6 +56,10 @@ def install(self, spec, prefix):
...
+        if not sys.platform == 'darwin':
+            config_args.extend(['LDFLAGS=-lrt'])
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/osu-micro-benchmarks/package.py]: You can use append and you won't need to create a new list.

Comment (pramodk) [inline var/spack/repos/builtin/packages/osu-micro-benchmarks/package.py]: done

File: var/spack/repos/builtin/packages/osu-micro-benchmarks/package.py (hunk 1 above)
Comment (pramodk) [inline var/spack/repos/builtin/packages/osu-micro-benchmarks/package.py]: fixed

**Pull request merged**